README.md

```diff
@@ -36,24 +36,24 @@ Seneca()
   // Get API keys using the seneca-env plugin
   .use('env', {
     var: {
-      $TANGOCARD_APIKEY: String,
-      $TANGOCARD_USERTOKEN: String,
+      $CHECKLYHQ_APIKEY: String,
```

Comment:
update these docs as well to match the checkly vars

Reply:
Ok, done